src/main/java/com/feedzai/commons/sql/abstraction/engine/impl/MySqlTranslator.java

```diff
@@ -282,7 +282,30 @@ public String translate(DbColumn c) {
 
     @Override
     public String translate(final Cast cast) {
-        throw new OperationNotSupportedRuntimeException("PDB does not support Cast in MySQL.");
+        final String type;
+
+        switch (cast.getType()) {
+            case BOOLEAN:
+            case INT:
+            case LONG:
+                type = "SIGNED INTEGER";
+                break;
+            case DOUBLE:
+                type = "DECIMAL(65,30)";
+                break;
+            case STRING:
+                type = "CHAR";
```

Comment:
Not VARCHAR?

Reply:
as strange as it may seem, in MySQL 5.7 (and v8.0 doesn't add much to this) we can only cast to:
BINARY
CHAR
DATE
DATETIME
DECIMAL
JSON
NCHAR
SIGNED [INTEGER]
TIME
UNSIGNED [INTEGER]

Reply:
(but it's like casting to a string, it gets converted with whatever characters it needs, doesn't get truncated)